inspec.yml

```diff
@@ -6,7 +6,7 @@ copyright_email: saf@groups.mitre.org
 license: Apache-2.0
 summary: "Inspec Validation Profile for Red Hat Enterprise Linux 7 STIG"
 version: 3.6.0
-inspec_version: ">= 4.0"
+inspec_version: ">= 4.26"
```

Comment:
Not sure why .26?

Reply:
I remember that the file had two instances of the inspec_version in this file - one saying >= 4.0 and one saying  >= 4.26. I saw warnings saying something to the effect of "you defined this variable twice" and chose the more conservative of the two listed. I am looking at the [changelog from InSpec](https://raw.githubusercontent.com/inspec/inspec/main/CHANGELOG.md), but do not know the specific reason for 4.26.